### DISCUSSION TITLE
How did I become a developer?

### DISCUSSION THREAD

**Discussion by ghost** [EddieHubCommunity/support#1827] [Show and tell] [answered]
How did I become a developer? What were the things that leveled me up? What were my pros?

This is a really useful tweet for beginners!

**Comment by ghost** [comment, marked as answer]
https://twitter.com/krishdevdb/status/1402580145051869190

**Reply by ashwinexe** [reply]
VS Code is one of the best for me, I've tried other IDEs too, but I eventually end up using VSCode somehow
